prov/verbs/src/fi_verbs.c

```diff
@@ -465,6 +469,25 @@ static int fi_ibv_get_param_bool(const char *param_name,
 	return 0;
 }
 
+static int fi_ibv_get_param_long(const char *param_name,
+				 const char *param_str,
+				 long *param_default)
+{
+	int ret;
+	long param;
+
+	ret = fi_ibv_param_define(param_name, param_str,
+				  FI_PARAM_LONG,
+				  param_default);
+	if (ret)
+		return ret;
+
+	if (!fi_param_get_long(&fi_ibv_prov, param_name, &param))
+		*param_default = param;
+
+	return 0;
+}
+
```

Comment:
In order to keep changes to the core in a separate patch, verbs specific changes can be moved to the patch that adds `FI_VERBS_MR_MAX_CACHED_SIZE ` env variable

Reply:
I'e moved these changes into a separate patch